2018-07-25-Preparing-ECS-to-EKS/slides.md

```diff
@@ -0,0 +1,140 @@
+class: center, middle
+
+# ECS to EKS
+## Preparing for the journey
+
+by Drew MacInnis
+
+---
+# What to pack?
+
+* an AWS account
+* an existing project running on ECS
+* terraform using terraform modules 
+  * backpack full of [terraform-aws-modules](https://github.com/terraform-aws-modules)
+* plenty of food and water
+
+<img src="images/mec-nalgene-bottle.png" style="opacity: 0.75; position: absolute; bottom: 50px; right: 0px"></img>
+ 
+---
+# Where are we starting from?
+
+* not an expert in ECS or EKS/k8s...
+* working with a project with microservices running in ECS
+* using a simplified environment for experimentation:
+  * [github.com/drmdrew/ecs-eks-demo][1]
```

Comment:
In the live slides, this link points to https://aws.amazon.com/blogs/opensource/vpc-cni-plugin-v1-1-available/
Probably something I did, just a heads up. The other links are fine.